enclave/core/sgx/enter.S

```diff
@@ -119,6 +119,23 @@ oe_enter:
     mov %rsp, %rbp
 
 .call_function:
+    // Set the MXCSR according to the Linux x86_64 ABI
+    mov $ABI_MXCSR_INIT, %r10
+    push %r10
```

Comment:
Just occurred to me that push pop are not needed here and for fldcw
You could do
```asm
mov $ABI_MXCSR_INIT, (%rsp)
ldmxcsr (%rsp)
```

Reply:
Thanks for the suggestion, but I think it's clearer to keep the two inits apart rather than save a push by reusing `(%rsp)`